source/getting_started/on-vm.rst

```diff
@@ -13,68 +13,26 @@ Prerequisites
 *************
 
 * Linux\* kernel version 5.0.0 or above running as the host OS
-* USB flash drive
-* At least 16GB free disk space on the host
 
 Prepare the host environment
 ****************************
 
+For simpifing the preparation work, the helper script :file:`setup_host.sh` is
+provided.
 The host device that launches the virtual machine requires Linux kernel
 version 5.0.0 or above running as the host OS. Complete the following
-instructions to set up `Clear Linux OS <https://clearlinux.org/>`_ 30870 as
-the host device. Any later Clear Linux OS version should also work.
+instructions to set up `Clear Linux OS <https://clearlinux.org/>`_ 30870 or
+`Ubuntu OS <https://ubuntu.com/>`_ 18.04.3 as the host device.
+Any later OS version should also work.
+Questions will be asked before changes happening during the installation
+process, answer 'y' for them.
```

Comment:
During the installation, you will be prompted by some questions to confirm the changes to the packages, it's safe to respond 'y' to all of them.